tests/test_ops.py

```diff
@@ -1417,6 +1449,10 @@ def test_mathtransform():
         # TODO: Div just gives "TypeError: unsupported operand type(s) for /: 'list' and 'list'" event through other ops work :/
         # {"opName": "div", "outName": "emptyArrayTests/div/rank1", "varShapes":[[0], [0]], "varTypes":["float32", "float32"], "varInit":["empty","empty"], "dtype":tf.float32},
         # {"opName": "div", "outName": "emptyArrayTests/div/rank2", "varShapes":[[0,2], [0,2]], "varTypes":["float64", "float64"], "varInit":["empty", "empty"], "dtype":tf.float64},
+        #  {"opName": "div", "outName": "div", "varShapes":[[1,10], [1,10]], "varTypes":["float64", "float64"], "varInit":["uniform", "uniform"], "dtype":tf.float64},
+
+        #{"opName": "div_no_nan", "outName": "div_no_nan", "varShapes":[[1,10], [1,10]], "varTypes":["float32", "float32"], "varInit":["uniform","uniform"], "dtype":tf.float32},
+        #{"opName": "div_no_nan", "outName": "div_no_nan", "varShapes": [[1, 10], [1, 10]],"varTypes": ["float64", "float64"], "varInit": ["uniform", "uniform"], "dtype": tf.float64},
```

Comment:
Docs and ops.proto aren't clear - but I would expect this op to support broadcasting.